tests/ignite/contrib/metrics/regression/test_wave_hedges_distance.py

```diff
@@ -84,7 +84,7 @@ def get_test_cases():
             _test(y_pred, y, batch_size)
 
 
-def _test_distrib_compute(device):
+def _test_distrib_compute(device, tol=1e-5):
```

Comment:
Do we really need to change tolerence for xla ?

Reply:
I did that just in case of we needed to reduce the tolerence for xla as we almost do (we did that in `FractionalBias`), what do you think?

Reply:
I'd prefer to adjust that if there is already a precision failure

Reply:
Sure! i will keep the tolerence `1e-5` to see what's going to happen

Reply:
IMO if `tol` is not currently needed (because it works), we should remove it, even if it could help. It is necessary to avoid under-quality but also over-quality in software engineering 😉

Reply:
Thanks for this @sdesrozis, appreciate your advice!

Reply:
@KickItLikeShika can you please remove `tol` at all